src/Ng/Ng.csproj

```diff
@@ -415,6 +418,34 @@
     </BootstrapperPackage>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="binaries\azcopy\AzCopy.exe">
```

Comment:
Are we allowed to check in AzCopy like this? From the website it looks like only an MSI. Now we're checking in a copy to our repo with no license.